contactmps/static/javascript/base.js

```diff
@@ -15,18 +15,23 @@ $(window).on('load', function() {
 // social-sharing
 $('.fb-share').on('click', function(e) {
   e.preventDefault();
-  var url = encodeURIComponent($(this).data('url'));
+  var $this = $(this),
+      url = encodeURIComponent($this.data('url')),
+      msg = encodeURIComponent($this.data('message') || '');
 
-  window.open("https://www.facebook.com/sharer/sharer.php?u=" + url, "share", "width=600, height=400, scrollbars=no");
-    ga('send', 'social', 'facebook', 'share', url);
+  window.open("https://www.facebook.com/sharer/sharer.php?u=" + url + "&quote=" + msg, "share", "width=600, height=400, scrollbars=no");
```

Comment:
is this working as you'd expect? secretballot was using base-ajax.js last I checked